lib/puppet/module_tool/skeleton/templates/generator/Gemfile

```diff
@@ -1,7 +1,14 @@
-source 'https://rubygems.org'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
```

Comment:
Hi @adreyer 

This doesn't have to do with internal use vs forge modules. Its purpose is to speed up build pipelines by allowing you to specify your internal mirror of rubygems.org as opposed to connecting to the internet to download them each time you test. It is already standard with Puppet Labs' own modules.